crates/core/src/node/in_memory_ext.rs

```diff
@@ -284,10 +284,10 @@ impl InMemoryNode {
             .ok_or_else(|| anyhow!("code must be 0x-prefixed"))?;
         let bytecode = hex::decode(code_slice)?;
         zksync_types::bytecode::validate_bytecode(&bytecode).context("Invalid bytecode")?;
-        tracing::info!(
-            ?address,
-            bytecode_hash = ?BytecodeHash::for_bytecode(&bytecode).value(),
-            "set code"
+        tracing::debug!(
+            "set code: address={:?}, bytecode_hash={:?}",
+            address,
+            BytecodeHash::for_bytecode(&bytecode).value()
```

Comment:
The order was correct before, this is how you output structured logs/events in tracing. See [docs](https://docs.rs/tracing/latest/tracing/)